### PR TITLE
added sorting landing page

### PR DESCRIPTION
**Pull request by adrian-taay**
sorting landing page

for further revisions in design once all components are in place.

### CODE REVIEW

File: public/akshar-dave-lXIpMj1AFUI-unsplash.jpg
Comment (miracle09110) [inline public/akshar-dave-lXIpMj1AFUI-unsplash.jpg:1]: can you rename this file to something more readable

File: src/pages/sorting/Cards.css [hunk 1]
```diff
@@ -0,0 +1,59 @@
+.card {
```
Comment (miracle09110) [inline src/pages/sorting/Cards.css]: if we're gonna use raw CSS, can you please use module.

Just rename this file to `cards.module.css`

then import on the jsx file

```
import style from "./cards.module.css"
```

example like what james did

https://github.com/miracle09110/concept-visualizer/blob/1ad610bc087631e53d388f2f5f9baf2afb1d7d71/src/pages/landingPage/explore/Explore.view.jsx

File: src/pages/sorting/Cards.jsx [hunk 2]
```diff
@@ -0,0 +1,27 @@
+import PropTypes from "prop-types";
+import { FaAngleRight } from "react-icons/fa6";
+import "./Cards.css";
```
Comment (miracle09110) [inline src/pages/sorting/Cards.jsx]: please use module style for importing css.

Can we also separate each component to a folder. Like
```
Cards
   -> Cards.jsx
   -> cards.module.css
SortingPage
    -> SortingPage.jsx
    -> sortingPage.module.css
```

File: src/pages/sorting/Cards.jsx
Comment (miracle09110) [inline src/pages/sorting/Cards.jsx:1]: Move this component to `component` folder, since it is not a page

File: src/pages/sorting/Cards.jsx (hunk 2 above)
Comment (miracle09110) [reply src/pages/sorting/Cards.jsx]: Also please use `.view.jsx` for this file to make it Cards.view.jsx`

File: src/pages/sorting/Cards.jsx [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+  );
+}
+
+Cards.propTypes = {
+  title: PropTypes.string,
+  subtitle: PropTypes.string,
+  color: PropTypes.string,
+};
+
+export default Cards;
```
Comment (miracle09110) [inline src/pages/sorting/Cards.jsx]: Sorry a lot of comments, expected format should be like this
```
src
   components
      Cards
          Cards.view.jsx
          cards.module.css
    pages
       SortingPage
           SortingPage.view.jsx
            sortingpage.module.css
```

**Review by miracle09110 — CHANGES_REQUESTED** [review]
please see review

File: public/selection-sort.png
Comment (miracle09110) [inline public/selection-sort.png:1]: this image doesn't seem to be used this PR, can we remove?

**Comment by adrian-taay** [comment]
subject for changes in design

**Review by miracle09110 — DISMISSED** [review]

**Review by miracle09110 — DISMISSED** [review]

**Review by miracle09110 — APPROVED** [review]

**Review by miracle09110 — APPROVED** [review]

**Pull request merged**